makefile

```diff
@@ -170,11 +170,15 @@ $(buildDir)/build-cross-compile:scripts/build-cross-compile.go makefile
 $(buildDir)/make-tarball:scripts/make-tarball.go $(buildDir)/render-gopath
 	$(vendorGopath) go build -o $@ $<
 dist:$(buildDir)/dist.tar.gz
+# don't cross-compile agents for a single-architecture environment, like in testing
+dist-single-arch:$(buildDir)/dist-single-arch.tar.gz
```

Comment:
I think it would be interesting to, rather than have a bunch of different subtly different tarballs, it'd be better to do something along the lines of EVG-1614 and EVG-1979, where: 

- if you specify a RACE_DETECTOR envvar, the binaries just get the race detector, and we can get rid of these crazy alternate named race binaries which we can't even really use,
- If everything is in one binary, then we can more easily run local testing on a single platform from a single binary. 

Both of these tickets are in the next sprint.